components/Sidebar/HeaderTab.tsx

```diff
@@ -7,32 +7,39 @@ import {
 } from '@mui/material'
 import { Dispatch, SetStateAction, memo, useState } from 'react'
 import ArrowDropDownIcon from '@mui/icons-material/ArrowDropDown'
-import SubheaderTab from './SubheaderTab'
 import { SidebarData } from 'constants/sidebarData'
 import { useRouter } from 'next/router'
+
 import { DRAWER_WIDTH } from '.'
-import { MR_GRAY_1 } from 'styles/theme'
+import SubheaderTab from './SubheaderTab'
+
+const SELECTED_TAB_COLOR = '#FAB800'
+const UNSELECTED_TAB_COLOR = 'white'
+const SELECTED_FONT_COLOR = 'white'
+const UNSELECTED_FONT_COLOR = 'black'
+const SELECTED_ICON_COLOR = 'white'
+const UNSELECTED_ICON_COLOR = '#009DD7'
 
 const HeaderTab = ({
+  isSidebarOpen,
   headerData,
   tabSelected,
   setTabSelected,
 }: {
+  isSidebarOpen: boolean
   headerData: SidebarData
   tabSelected: string
   setTabSelected: Dispatch<SetStateAction<string>>
 }) => {
   const [expanded, setExpanded] = useState<boolean>(false)
   const hasSubheaders: boolean = headerData.subheaders.length > 0
   const router = useRouter()
-  // Being re-rendered twice, even on opening and closing, not sure why
-  // console.log('Rendered')
 
   return (
     <>
       <ListItem key={headerData.title} disablePadding>
```

Comment:
for onHover, we should have some shadow to show u are selecting it. something like this https://mui.com/material-ui/react-drawer/#mini-variant-drawer